pages/docs/index.js

```diff
@@ -277,6 +277,16 @@ export default function DocsIndex() {
       title: "Extra",
       description: "More information",
       pages: [
+        {
+          name: "Profile Tips",
+          path: "/docs/profile-tips",
+          description:
+          "Tips on making your LinkFree profile stand out.",
+          category: { 
+            name: "Intermediate",
+            color: "bg-orange-100 text-orange-800",
```

Comment:
reflecting the colour associated with `beginner`:
```suggestion
            color: "bg-green-100 text-green-800",
```